DataFormats/Common/interface/Ptr.h

```diff
@@ -182,14 +182,15 @@ namespace edm {
         WrapperBase const* prod = getter->getIt(core_.id());
         unsigned int iKey = key_;
```

Comment:
Is this variable needed anymore? At least it should be moved to just before where it is used.

Reply:
It is needed on line 196
https://github.com/cms-sw/cmssw/blob/029426d38df5f46617412976c1dba5f1bdcd680c/DataFormats/Common/interface/Ptr.h#L196
with the semantics "`key`, except if parent collection is not found and thinned collection is found, then the key to the thinned collection". I would have liked to remove it though.